code/pizzabot.js

```diff
@@ -1,7 +1,65 @@
 const vegetarian = "Vegetarian Pizza"
 const hawaiian = "Hawaiian Pizza"
 const pepperoni = "Pepperoni Pizza"
+const margherita = "Margherita Pizza"
 
 const pizzaPrice = 80
 
-//Put your Javscript code here:
+const pizzaVegImg = document.getElementById("veg")
+const pizzaHawImg = document.getElementById("haw")
+const pizzaPepImg = document.getElementById("pep")
+const pizzaMarImg = document.getElementById("mar")
+
+
+console.log(vegetarian, hawaiian, pepperoni, pizzaPrice)
+
+document.getElementById("menu").innerHTML = `Hey! Happy to serve your pizza. On our menu we have ${vegetarian}, ${hawaiian}, ${pepperoni}, and ${margherita}.`
+
+
+const calculateTotalCost = (orderQuantity) => {
+  return orderQuantity * pizzaPrice
+}
```

Comment:
Very nice :) Since we have only one line inside `calculateTotalCost `function, we can write it in "arrow function shorthand" :

`const calculateTotalCost = orderQty => orderQty * pizzaPrice;`

We can do that because in arrow functions we can omit curly brackets (`{}`) and `return` keyword if we only have one line of code inside.
For further reference on that subject, please check out https://www.w3schools.com/js/js_arrow_function.asp